### PR TITLE
PLANET-5190 Fix EN form acceptance test

### PR DESCRIPTION
**Pull request by sagarsdeshmukh**
related to [JIRA 5190](https://jira.greenpeace.org/browse/PLANET-5190)

### CODE REVIEW

File: tests/acceptance/ENFormCest.php [hunk 1]
```diff
@@ -277,20 +274,19 @@ public function createAnEnForm( AcceptanceTester $I ) {
...
 	 * @group enform
 	 * @group engaging-networks
-	 * @skip
 	 *
 	 * @param AcceptanceTester $I
 	 *
 	 * @throws \Codeception\Exception\ModuleException
 	 */
-	public function createAnEnBlock( AcceptanceTester $I ) {
+	public function createAnTestEnBlockFullWidth( AcceptanceTester $I ) {
```
Comment (mleray) [inline tests/acceptance/ENFormCest.php]: Not a huge deal, but I think there's a typo in the function name here? Shouldn't it be `createAndTestEnBlockFullWidth`?

File: tests/acceptance/ENFormCest.php [hunk 2]
```diff
@@ -301,90 +297,257 @@ public function createAnEnBlock( AcceptanceTester $I ) {
...
+	 *
+	 * @depends createAnEnForm
+	 * @group enform
+	 * @group engaging-networks
+	 *
+	 * @param AcceptanceTester $I
+	 *
+	 * @throws \Codeception\Exception\ModuleException
+	 */
+	public function createAnTestEnBlockFullWidthBG( AcceptanceTester $I ) {
```
Comment (mleray) [inline tests/acceptance/ENFormCest.php]: Same here: not a huge deal, but I think there's a typo in the function name here? Shouldn't it be `createAndTestEnBlockFullWidthBG`?

File: tests/acceptance/ENFormCest.php [hunk 3]
```diff
@@ -301,90 +297,257 @@ public function createAnEnBlock( AcceptanceTester $I ) {
...
 	 * @group enform
 	 * @group engaging-networks
-	 * @skip
 	 *
+	 * @param AcceptanceTester $I
+	 *
+	 * @throws \Codeception\Exception\ModuleException
 	 */
-	public function fillAnEnBlock( AcceptanceTester $I ) {
+	public function createAnTestEnBlockSideStyle( AcceptanceTester $I ) {
```
Comment (mleray) [inline tests/acceptance/ENFormCest.php]: Same here: not a huge deal, but I think there's a typo in the function name here? Shouldn't it be `createAndTestEnBlockSideStyle `?

**Review by mleray — COMMENTED** [review]
Looks good 👍 Just a few minor comments!

**Comment by sagarsdeshmukh** [comment]
~~The EN tests are passed on my local(screenshot attached below) -~~
image

~~But failing on [CI pipeline](https://app.circleci.com/pipelines/github/greenpeace/planet4-base-fork/423/workflows/1528141e-4a59-4cc3-8eb8-81062bcc0636/jobs/5010/steps).~~

~~What I understood from the errors and logs, is that the cache which I set from [CLI command](https://github.com/greenpeace/planet4-base-fork/pull/79/files#diff-fb64434d552629bc66569d4420564647R159) is not work as indented.~~

~~after adding a cache the "Available fields", "Available Questions" and "Available Opt-ins" are populate on backend [enform builder page](https://5010-131222745-gh.circle-artifacts.com/0/planet4-docker-compose/artifacts/codeception/record_5ef21d0f3c2dc_ENFormCest.php_createAnEnForm/index.html). but its not happening somehow.~~ :(

Never mind, Issue solved by adding the [dummy engaging networks api keys](https://github.com/greenpeace/planet4-base-fork/pull/79/files#diff-fb64434d552629bc66569d4420564647R130)

**Comment by sagarsdeshmukh** [comment]
Hi @mleray & @comzeradd , The EN form acceptance tests are ready and working in CI pipeline. could you please have a look once.

**Review by comzeradd — APPROVED** [review]
Nice work :100:

**Pull request merged**